hazelcast/src/main/java/com/hazelcast/core/IMap.java

```diff
@@ -73,6 +74,25 @@
  * </p>
  * <p>This class does <em>not</em> allow <tt>null</tt> to be used as a key or value.</p>
  *
+ * <p>Entry Processing</p>
+ * The following operations are lock-aware, since they operate on a single key only.
+ * If the key is locked the EntryProcessor will run until it acquires the lock.
```

Comment:
" If the key is locked the EntryProcessor will *wait* until it acquires the lock." ?